tests/conftest.py

```diff
@@ -127,6 +127,11 @@ def setup_install_pkgs(ansible_module):
     This fixture is used in test_positive_check_hotfix_installed_with_hotfix and
     test_positive_check_hotfix_installed_without_hotfix of test_health.py
     """
+    setup = ansible_module.yum(
+        name='fio',
+        state='present')
```

Comment:
Added this because testfm has issue with packages-install which prevents it from installing required packages for foreman-maintain to work properly. So here I am making sure that the required packages are already present or not.